specification/security/resource-manager/Microsoft.Security/preview/2025-10-01-preview/operationResults.json

```diff
@@ -0,0 +1,103 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
+    "description": "API spec for Microsoft Defender for Cloud operation results.",
+    "version": "2025-10-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/locations/{location}/operationResults/{operationId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get operation result - Completed (200)": {
+            "$ref": "./examples/OperationResults/GetOperationResult.json"
+          },
+          "Get operation result - In Progress (202)": {
+            "$ref": "./examples/OperationResults/GetOperationResultInProgress.json"
+          },
+          "Get operation result - No Content (204)": {
+            "$ref": "./examples/OperationResults/GetOperationResultNoContent.json"
+          },
+          "Get operation result - Error": {
+            "$ref": "./examples/OperationResults/GetOperationResultError.json"
+          }
+        },
+        "tags": [
+          "OperationResults"
+        ],
+        "description": "Returns operation results for long running operations executing DELETE or PATCH on the resource.",
+        "operationId": "OperationResults_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/OperationIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v6/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the operation result."
+          },
+          "202": {
```

Comment:
When reading the operation resource entity, the status code should always be 200. The response payload indicates if the async operation is still running or has completed (successfully or with a failure).

See here: 
https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/async-api-reference.md#asynchronous-operations

Reply:
Please check the table at https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/async-api-reference.md#example-flow
operation results has 202 and Location, RetryAfter support. tried to map it again and updated the PR accordingly